t/admin/routes2.t

```diff
@@ -197,6 +197,7 @@ GET /t
             res.node.key = nil
             res.node.value.create_time = nil
             res.node.value.update_time = nil
+            res.node.value.id = nil
```

Comment:
Can we check if id is not nil?

Reply:
Thanks for your review firstly. the id (like the key) is generated by the server, and different for every test.

Reply:
We can just check if it is not nil

Reply:
okay, I will add the check, thanks!

Reply:
the check is done. @spacewander